wp-content/themes/wp-starter/src/styles/base/global.css

```diff
@@ -3,29 +3,47 @@
 		@apply !hidden;
 	}
 
-	ul {
-		@apply list-disc;
+	ul.wp-block-list {
+		@apply list-disc pl-32;
 
 		li {
-			@apply list-inside;
+			@apply list-outside;
 		}
 	}
 
-	ol {
-		@apply list-decimal;
+	ol.wp-block-list {
+		@apply list-decimal pl-32;
 
 		li {
-			@apply list-inside;
+			@apply list-outside;
 		}
 	}
+
+	a {
+		@apply underline;
+	}
+
+	p:not(:empty):not(:last-child) {
+		@apply mt-24;
+	}
 }
 
 @layer components {
 	/*
-		This adds a bit of space between blocks on a page.
+		Adjust this to add space between blocks on a page.
 		Should be adjusted or removed as needed per project.
 	*/
 	.entry-content > *:not(:first-child) {
-		@apply mt-12;
+		@apply mt-0;
+	}
+
+	.entry-content > *:not(.alignfull),
+	.alignfull.is-layout-constrained > .acf-block__container {
+		max-width: theme('contentBase');
+		@apply mx-auto;
+	}
+
+	.entry-content > .alignfull {
```

Comment:
Needed to add the `>` otherwise we could not nest blocks inside of other blocks.